Week2/todo_app/app.js

```diff
@@ -0,0 +1,194 @@
+// This is the connector (also known as driver)
+// that we can use to connect to a MySQL process
+// and access its databases.
+const mysql = require('mysql');
+
+class TodoModel {
+    constructor(dbConnection) {
+        this.dbConnection = dbConnection;
+    }
+
+    // Loads all the TODOs in the database
+    load(callback) {
+        const selectTodoItems = "SELECT * FROM todo_items";
+        this.dbConnection.query(selectTodoItems, function (err, results, fields) {
+            if (err) {
+                callback(err);
+                return;
+            }
+
+            callback(null, results);
+        });
+    }
+
+    create(userId, description, callback) {
+        // Write code and query to create a new TODO item
+        const createTodoItem = `insert into todo_items (text,user_id) values(` + mysql.escape(description) + `,` + mysql.escape(userId) + `);`;
```

Comment:
You can use the `${}` trick to make it easier to read ;)

Reply:
I used it in / Program.js / file 👍